tests/test_redis_config.py

```diff
@@ -0,0 +1,166 @@
+from unittest import TestCase, skip
+from subprocess import run, PIPE
+from uuid import uuid4
+import random
+import time
+
+from Jumpscale import j
+from parameterized import parameterized
+from loguru import logger
+from redis import ResponseError
+
+LOGGER = logger
```

Comment:
u shouldn't need that logger if your client is in the tests folder of the clients